app/controllers/users_controller.rb

```diff
@@ -18,6 +23,10 @@ def create
 
   private
 
+  def connection
+    Faraday.get('https://api.github.com/users/davidttran/repos?sort="updated"')
```

Comment:
Hard coding the URL for now because we don't have a way to assign any users a GH token (through the app)